Sources/Core/Extensions/URLExtension.swift

```diff
@@ -19,6 +19,13 @@ extension URL {
     }
 
     private static func make(from string: String, relativeTo baseURL: URL) -> URL {
+        let range = NSRange(location: 0, length: string.utf16.count)
```

Comment:
Maybe this will always be correct, but I believe the idiomatic way to do this is `NSRange(string.startIndex ..< string.endIndex, in: string)`.